python/lsst/dax/imgserv/dispatch.py

```diff
@@ -22,14 +22,12 @@
 """
 This module implements the API dispatch logic.
 
-@author: Kenny Lo, SLAC
-
 """
 
 import os
 import json
 
-from .image_v1 import Image
+from .image import Image
 from .hashutil import Hasher
 
 
```

Comment:
You don't need to call jason_api.close(), with will close it for you automatically (sorry don't know how to leave a comment on line 45 which didn't change.

Reply:
Good catch.